src/main/java/com/bobocode/svydovets/annotation/bean/processor/AutoSvydovetsSetterBeanProcessor.java

```diff
@@ -0,0 +1,68 @@
+package com.bobocode.svydovets.annotation.bean.processor;
+
+import com.bobocode.svydovets.annotation.annotations.AutoSvydovets;
+import com.bobocode.svydovets.annotation.annotations.Qualifier;
+import com.bobocode.svydovets.annotation.bean.factory.BeanFactory;
+import com.bobocode.svydovets.annotation.exception.BeanException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+/**
+ * Implementation of the {@link BeanProcessor} interface. Executes processing of
+ * {@link AutoSvydovets} marked setters in beans. Such fields will be injected as a
+ * dependency, retrieved from the context.
+ * <p>
+ * If no unique bean is present - processor will try to inject bean based on
+ * {@link Qualifier} value.
+ *
+ * @see BeanProcessor
+ * @see AutoSvydovets
+ * @see Qualifier
+ */
+public class AutoSvydovetsSetterBeanProcessor implements BeanProcessor {
```

Comment:
Maybe we should use only one processor? 
We can check fields and methods by every `beanType`, and don't go through the Map twice (or more)